edit-todo

Jeffersonlpu opened the pull request:
The possibility to change the todo was enabled.

Jeffersonlpu committed:
edit-todo

The possibility to change the todo was enabled.

jhrodriguesit commented on src/App.tsx:
```diff
@@ -18,9 +18,21 @@ const App = () => {
   };
 
   useEffect(() => {
+    console.log("Updated items:", items);
```
```suggestion
```
Could you please remove this debug line?

Jeffersonlpu replied:
Done!

jhrodriguesit commented on src/components/Todo/Todo.tsx:
```diff
@@ -54,8 +70,17 @@ const Todo = ({
+              backgroundColor: getBackGroundColor(),
+            }}
+            type="text"
+            value={text?.toString()}
```
```suggestion
value={text}
```
If the text type is string I don't think you need to parse the value toString.

Jeffersonlpu committed:
remove-null

The const items was changed; now it only accept strings.

Jeffersonlpu committed:
remove-and-rename-hasNull

The property name was changed from hasEmptyString, and the null check was also was removed from handleRemoveTodo.

Jeffersonlpu committed:
add-auto-focus

The autoFocus attribute was added to the input field of Todo.
